WordPress/Classes/ViewRelated/Me/AppSettingsViewController.swift

```diff
@@ -350,6 +362,81 @@ fileprivate struct MediaSizingRow: ImmuTableRow {
     }
 }
 
+fileprivate struct VideoSizingRow: ImmuTableRow {
+    typealias CellType = MediaSizeSliderCell
+
+    static let cell: ImmuTableCell = {
+        let nib = UINib(nibName: "MediaSizeSliderCell", bundle: Bundle(for: CellType.self))
+        return ImmuTableCell.nib(nib, CellType.self)
+    }()
+    static let customHeight: Float? = CellType.height
+
+    let title: String
+    let value: MediaSettings.VideoResolution
+    let onChange: (MediaSettings.VideoResolution) -> Void
+
+    let action: ImmuTableAction? = nil
+
+    func configureCell(_ cell: UITableViewCell) {
+        let cell = cell as! CellType
+        cell.model = VideoSizeModel.default
+        cell.title = title
+        cell.value = value.intValue
+        cell.onChange = { (value) in
+            self.onChange(MediaSettings.VideoResolution.videoResolution(from: value))
+        }
+        cell.selectionStyle = .none
+
+        cell.minValue = MediaSettings.VideoResolution.size640x480.intValue
```

Comment:
Let's use `VideoSizeModel.default.minValue` and `VideoSizeModel.default.maxValue` here.